webapp/channels/src/components/integrations/confirm_integration/confirm_integration.tsx

```diff
@@ -356,19 +356,19 @@ const ConfirmIntegration = ({team, location, commands, oauthApps, incomingHooks,
                 <p className='word-break--all'>
                     <FormattedMessage
                         id='add_outgoing_oauth_connection.token_url'
-                        defaultMessage='<b>Token URL</b>: `{url}`'
+                        defaultMessage='<b>Token URL</b>: {url}'
```

Comment:
You'll also need to update `i18n/en.json` for these changes to the `defaultMessage`s. The current i18n tooling unfortunately doesn't sync changes to the `defaultMessage` with that file

Reply:
I checked all the i18n files and I couldn't find any with the backticks in it.. strange